src/worldstate/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(worldstate)
+
+## Compile as C++11, supported in ROS Kinetic and newer
```

Comment:
In general, we try and cut down `CMakeLists.txt` files so they're short and readable (a fully commented version is always available in `sample_package`). For an example of this, see [here](https://github.com/UBC-Snowbots/Snowflake/blob/master/src/sb_pointcloud_processing/CMakeLists.txt)